flow/designs/nangate45/black_parrot/config_hier_tmp.mk

```diff
@@ -0,0 +1,4 @@
+include $(dir $(DESIGN_CONFIG))/config_hier.mk
+
+export FLOW_VARIANT = hier_tmp
+unexport RTLMP_FLOW
```

Comment:
I see a couple of tmp files that are already on master. What is the purpose of these tmp files?

Reply:
the config_hiier_tmp file  are running the hierarchical netlist flow (yosys run with hierarchical verilog) through Triton Macro Placer. This is for apples to apples comparison for RTLMP vs TMP for  same design input (RTL, SDC, FP outline)

Reply:
I see, I assumed tmp as for temp/temporary.